README.md

```diff
@@ -97,6 +97,8 @@ For more information see [CommandRef](https://fhem.de/commandref.html#update) or
 ### FHEM SVN
 You can also get the required files directly from the FHEM SVN with typing the following command into FHEM's command-line:\
 ```{ Svn_GetFile('contrib/RHASSPY/10_RHASSPY.pm', 'FHEM/10_RHASSPY.pm') }```
+and (if you e.g. want RHASSYP to respond in German):
```

Comment:
if you e.g. → e.g. if you want
RHASSYP → RHASSPY
respond in German → respond in german